src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java

```diff
@@ -372,4 +372,14 @@ public boolean isSectionChanged(StudentAttributes originalStudentAttribute) {
     public boolean isTeamChanged(StudentAttributes originalStudentAttribute) {
         return this.team != null && !this.team.equals(originalStudentAttribute.team);
     }
+    
+    /**
+     * Checks whether the edit form of student has changed the email value.
```

Comment:
I think better to do it as a firsttimer issue (we are short of those anyway :-p)